### ISSUE TITLE
[BUG] `module 'pkgutil' has no attribute 'ImpImporter'` error for command `trakts init` with python 3.12

### ISSUE THREAD

**Issue by soredake** [iamkroot/trakt-scrobbler#272]
**Describe the bug**

A clear and concise description of what the bug/error is.

**Desktop (please complete the following information):**
 - OS and Version: Windows 11 Pro
 - Python Version: Python 3.12

**To Reproduce**

1. Install python 3.12
2. Install trakt-scrobbler with pipx
3. Oh no, error!
4. See error in log file

**Log file**

<details><summary>Click to see log contents</summary> 
<p>

```
❯ trakts init
This will guide you through the setup of the scrobbler.
If you wish to quit at any point, press Ctrl+C or Cmd+C

  AttributeError

  module 'pkgutil' has no attribute 'ImpImporter'

  at AppData\Local\pipx\pipx\venvs\trakt-scrobbler\Lib\site-packages\pkg_resources\__init__.py:2191 in <module>
      2187│     dist_groups = map(find_distributions, resolved_paths)
      2188│     return next(dist_groups, ())
      2189│
      2190│
    → 2191│ register_finder(pkgutil.ImpImporter, find_on_path)
      2192│
      2193│ if hasattr(importlib_machinery, 'FileFinder'):
      2194│     register_finder(importlib_machinery.FileFinder, find_on_path)
      2195│



2024-01-09 11:32:34,853 - DEBUG - MainThread - notifier - Notifications enabled for categories: exception, misc, scrobble.pause, scrobble.resume, scrobble.start, scrobble.stop, trakt
```
</p>
</details>

**Comment by iamkroot** [comment]
Already fixed in #257

**Comment by iamkroot** [comment]
Please search the existing issues before creating new ones,
